lua/nvim-treesitter/highlight.lua

```diff
@@ -13,6 +13,7 @@ local hlmap = vim.treesitter.TSHighlighter.hl_map
 hlmap.error = "Error"
 hlmap["punctuation.delimiter"] = "Delimiter"
 hlmap["punctuation.bracket"] = "Delimiter"
+hlmap["punctuation.special"] = "Delimiter"
```

Comment:
You might want to add this to the `CONTRIBUTING.md` file too.

Reply:
Also add a highlight `punctuation.bracket` for normal `{` and `}` (e.g. for dicts)